new/LIT6821HomEphrEpiphany.xml

```diff
@@ -0,0 +1,75 @@
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘቅዱስ፡ ወብፁዕ፡ ኤፍሬም፡ በእንተ፡ ጥምቀቱ፡ ለእግዚእነ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān za-qǝddus wa-bǝḍuʿ ʾEfrem baʾǝnta ṭǝmqatu la-ʾǝgziʾna</title>
+                <title xml:lang="en" corresp="#t1">Homily by the holy and blessed Ephrem on the baptism of our Lord</title>
+                <title xml:lang="en" xml:id="t2">Homily on Epiphany attributed to Ephrem</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This Homily is attested uniquely in <ref type="mss" corresp="EMML7015"/>, where it is attributed to <persName ref="PRS3827Ephrem"/>. </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2023-04-26">Created entity from data provided by Aaron Butts</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                    <listBibl type="secondary">
+                        <bibl><ptr target="bm:ButtsErho2018Jacob"/><citedRange unit="page">53</citedRange></bibl>
+                    </listBibl>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6821HomEphrEpiphany" passive="EMML7015"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT6821HomEphrEpiphany" passive="PRS3827Ephrem"/>
+                    <relation name="dcterms:creator" active="LIT6821HomEphrEpiphany" passive="PRS5655Jacobof"/>
```

Comment:
I would suggest adding to the abstract some explanation for the presence of Jacob.